fvm/src/kernel/default.rs

```diff
@@ -296,6 +296,9 @@ where
         self.call_manager
             .charge_gas(self.call_manager.price_list().on_block_open_base())?;
 
+        self.call_manager
+            .charge_gas(self.call_manager.price_list().on_extern())?;
```

Comment:
Personally, I'd just charge these together. I.e., make `on_block_open_base` add-in the extern cost. That way we don't accidentially _not_ charge the extern cost in some cases.

Reply:
Hm, in fact, I think it might be better that way because when we remove the extern cost (implementing the node-provided ring buffer idea for randomness), we can contain the gas charge change to the PriceList. (Otherwise we would have to do network version detection here to charge conditionally).

Reply:
Done.